go/common/grpc/grpc.go

```diff
@@ -494,6 +494,10 @@ func NewServer(config *ServerConfig) (*Server, error) {
 
 // Dial creates a client connection to the given target.
 func Dial(target string, opts ...grpc.DialOption) (*grpc.ClientConn, error) {
+	if len(target) > 0 && target[0] == '/' {
```

Comment:
This shouldn't be in the common gRPC code, rather it should be part of the gRPC wrapper for CLI tools, e.g. `go/oasis-node/cmd/common/grpc`.